lib/tzdata/data_loader.ex

```diff
@@ -4,8 +4,15 @@ defmodule Tzdata.DataLoader do
   require Logger
   # Can poll for newest version of tz data and can download
   # and extract it.
-  @download_url "https://data.iana.org/time-zones/tzdata-latest.tar.gz"
-  def download_new(url \\ @download_url) do
+  def download_url do
+    Application.fetch_env!(:tzdata, :download_url)
```

Comment:
It seems like this should be optional and default to the latest.

```suggestion
    Application.get_env(:tzdata, :download_url, @download_url)
```